package.json

```diff
@@ -17,23 +17,33 @@
     "@types/google-apps-script": "^0.0.35"
   },
   "devDependencies": {
-    "@google/clasp": "^1.7.0",
+    "@google/clasp": "^2.0.1",
     "del": "3.x",
+    "directory-exists": "latest",
     "exec-sh": "0.3.x",
     "gulp": "^4.0.0",
     "gulp-changed": "latest",
     "gulp-rename": "latest",
     "gulp-replace": "latest",
-    "jest": "^24.0.0",
-    "yargs": "^12.0.5",
-    "directory-exists": "latest"
+    "jest": "^24.1.0",
+    "yargs": "^12.0.5"
   },
   "scripts": {
     "test": "jest"
   },
   "jest": {
     "roots": [
       "<rootDir>/test"
+    ],
+    "modulePaths": [
+      "<rootDir>"
+    ],
+    "moduleFileExtensions": [
+      "gs",
+      "js"
+    ],
+    "collectCoverageFrom": [
+      "src/**/*.gs"
```

Comment:
this just improves the coverage report so it doesnt include the unit tests in the coverage output.
Also setting modulePaths means we can use absolute paths in the requires which makes things easier as we move files around the directory structure.